source/features/forked-to.tsx

```diff
@@ -123,7 +117,7 @@ async function init(): Promise<void | false> {
 	}
 
 	// This feature only applies to users that have multiple organizations, because that makes a fork picker modal appear when clicking on "Fork"
-	const hasOrganizations = await elementReady('details-dialog[src*="/fork"]', {waitForChildren: false});
+	const hasOrganizations = select.exists('[data-ga-click*="show fork modal"]');
```

Comment:
This targets an attribute on the "Fork" button itself, so we shouldn't need the `elementReady`. Not sure if this is really limited to people in organizations though.